plugins/default/lib/disk_usage.py

```diff
@@ -1,35 +1,97 @@
 #!/usr/bin/env python
 import re
 import sys
+import json
+import argparse
 from subprocess import CalledProcessError
+from collections import Counter
 
-from openshift import oc
+import openshift
 import nagios
 
 
-check_disk_cmd = ("df", "--output=target,pcent,ipcent")
+check_disk_cmd = ("df", "--output=pcent,ipcent,target")
 # Example output:
 # /etc/hosts      19%    8%
-check_disk_output_pattern = re.compile(r"([\w/]+)\s+(\d+)%\s+(\d+)%")
+check_disk_output_pattern = re.compile(
+    r"\s*(\d+)%\s+(\d+)%\s+(.+)$")
```

Comment:
It wasn't me, it was autopip.  Not really ecstatic about its line length limit as it makes some of these things look really stupid.  Think we have excluded that check in the travis job, so I could put these lines back the way they were and run with the same skip.